### PR TITLE
feat: implement RTKInfo as a synthetic representation of the RTK functionality

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: src/RTKInfo.hpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        /** How many satellites are tracking carrier range (i.e. fixed phase) */
+        uint8_t satellites_with_carrier_range = 0;
+
+        /** Amount of RTCM data transmitted */
+        size_t tx = 0;
+        /** Amount of RTCM data received */
+        size_t rx = 0;
+
+        void update(PVT const& pvt);
+        void update(SatelliteInfo const& pvt);
```
Comment (caioaamaral) [inline src/RTKInfo.hpp]: `const& info`

File: src/RTKInfo.hpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        void update(RTCMReceivedMessage const& pvt);
```
Comment (caioaamaral) [inline src/RTKInfo.hpp]: `const& msg`

File: src/RTKInfo.hpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+#include <cstdint>
+#include <vector>
+
+#include <base/Time.hpp>
+
+
+namespace gps_ublox {
+    struct PVT;
+    struct RelPosNED;
```
Comment (caioaamaral) [inline src/RTKInfo.hpp]: should we have to add a `void update(RelPosNED const& rel_pose);`?

**Review by caioaamaral — APPROVED** [review]
LGTM!

File: src/RTKInfo.hpp (hunk 3 above)
Comment (doudou) [reply src/RTKInfo.hpp]: We will ... next PR

**Review by rodrigoleonello — APPROVED** [review]

**Pull request merged**